WordPress/Classes/ViewRelated/Blog/Site Picker/SitePickerViewController+SiteIcon.swift

```diff
@@ -180,7 +180,6 @@ extension SitePickerViewController {
         imageCropController.onCompletion = { [weak self] image, modified in
             self?.dismiss(animated: true)
             self?.uploadDroppedSiteIcon(image, completion: {
-                self?.blogDetailHeaderView.blavatarImageView.image = image
```

Comment:
I removed it for simplicity. It looks perfectly fine:

https://github.com/user-attachments/assets/e2680112-d20d-4237-a52e-a576e59cf6b2